src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 350;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: bold;
+  font-weight: 850;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin-top: 0;
+  padding-top: 20px;
+  font-family: Avenir, sans-serif;
+}
+
+.form {
+  padding: 0;
+  margin: 0;
+  border: none;
+}
+
+.label {
+  padding: 0;
+  margin: 0;
+  border: none;
+}
+
+.input {
+  width: 100%;
+  margin-top: 0;
+  margin-bottom: 20px;
+  height: 70px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 100%;
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 350;
+  background: #fff url(./images/Search.svg) no-repeat 26px;
+  background-size: 19px 19px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.inputSmall {
+  width: 100%;
+  margin-top: 0;
+  height: 42px;
+  padding-top: 9px;
+  padding-bottom: 10px;
+  padding-left: 33px;
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  line-height: 100%;
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 350;
+  background: #fff url(./images/Search.svg) no-repeat 12px;
+  background-size: 11px 11px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.inputSmall:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-style: bold;
+}
+
+.inputSmall:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-style: bold;
+}
+
+.input::placeholder {
```

Comment:
Almost all properties in .input::placeholder and inputSmall::placeholder are the same. You should put it under one rule set. And then change font-size for one of them